IDE/VSCode/Non-Modular/Gradle/hellofx/src/main/java/org/openjfx/FXMLController.java

```diff
@@ -0,0 +1,14 @@
+package org.openjfx;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.Label;
+
+public class FXMLController {
+
+    @FXML
+    private Label label;
+
+    public void initialize() {
+        label.setText("Hello, JavaFX");
```

Comment:
Unlike the modular version, this is missing the version information. Is there a reason for this?

Reply:
Ah, I copied it from https://github.com/openjfx/samples/blob/master/IDE/Eclipse/Non-Modular/Gradle/hellofx/src/main/java/org/openjfx/FXMLController.java#L12.

I will update to display the version information.